docs/configuration/trap-configuration.md

```diff
@@ -136,3 +136,11 @@ traps:
     annotations:
       annotation_key: annotation_value
 ```
+
+### Aggregate traps
+In case you want to see traps events collected as one event inside splunk you can enable it by setting `splunk.aggregateTrapsEvents`.
+Example:
+```yaml
+splunk:
```

Comment:
I think it is better to keep this under `traps` section instead of `splunk`, and even here in docs is under traps' configuration

Reply:
done

Reply:
don't you need a one newline before yaml section? does it render properly ?

Reply:
it renders normally
<img width="790" alt="image" src="https://github.com/splunk/splunk-connect-for-snmp/assets/139114006/7caa7827-1dc7-40af-8ddc-3d7d124fd38c">